docs/source/api/api_capabilities_id.rst

```diff
@@ -22,6 +22,8 @@ Manages a specific API capability.
 
 ``GET``
 =======
+.. deprecated:: 1.1
```

Comment:
I think this was actually deprecated in ATC version 4.0, not API version 1.1. Or was it in API version 1.5? That is our current API version, but the deprecation decision predates this notice, and was decided just prior to cutting a 4.x RC. This is the type of stuff that confuses me when a service has two versioning systems.

Reply:
right :( lets do ``` .. deprecated:: ATCv4``` I think that makes the most sense. Let me change all my current deprecate PRs